src/mumble/MumbleApplication.cpp

```diff
@@ -56,46 +60,47 @@ bool MumbleApplication::event(QEvent *e) {
 }
 
 #ifdef Q_OS_WIN
+/// gswForward forwards a native Windows keyboard/mouse message
+/// into GlobalShortcutWin's event stream.
+static void gswForward(MSG *msg) {
+	if (Global::g_global_struct) {
```

Comment:
Actually, you're right. I thought we accessed the GlobalShortcut engine through `g`. But we don't. So let's drop it.